### DISCUSSION TITLE
[Kavita] Scan epub with SP markers

### DISCUSSION THREAD

**Discussion by WeslyG** [Kareadita/Kavita#2710] [Ideas]
### Idea Description

I understand and am used to the concept of SP markers, for pdf files, it works fine, and is totally fine with me when books in the same series can be named in the correct order in a folder without having to load them into third party software (calibre etc) to edit them. Don't you think it would be a good idea to support a similar notation (prioritizing file name reading over metadata) regarding the content of metadata reading within an EPUB file. 
So that the experience of downloading files doesn't differ from file format to file format. Because right now it causes inconvenience when you upload some files one way and others another. 

Maybe I didn't read the documentation well, or maybe I didn't understand something, but I couldn't get epub to work with sp markers. 

I really understand this notation from Jellyfin/Plex and other multimedia players, I think it would be a good idea to follow a similar naming rule here as well. Especially since it works for a number of formats already. This will allow me to keep my files more clean in naming. 

I am using the docker version of kavita 0.7.14

### Duration of Using Kavita

a few months

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by majora2007** [comment]
I wasn't able to understand. Are you saying that Kavita isn't prioritizing SP marker for epubs and you are proposing that it does?

**Reply by WeslyG** [reply]
exactly

**Reply by majora2007** [reply]
I see. I have ran into this issue myself because with epubs, there is no way to force something into a special.

**Comment by majora2007** [comment]
This will be delivered on v0.8.2. Please see the wiki for how to get specials to work within book series.
